libcxx/utils/generate_feature_test_macro_components.py

```diff
@@ -1078,6 +1078,11 @@ def add_version_header(tc):
             "values": {"c++23": 202106},
             "headers": ["algorithm"],
         },
+        {
+            "name": "__cpp_lib_ranges_stride",
+            "values": {"c++23": 202207},
```

Comment:
Can you perhaps check whether there's been an update to this FTM since the patch started?